app/src/main/java/protect/card_locker/ManageGroupActivity.kt

```diff
@@ -232,6 +219,12 @@ class ManageGroupActivity : CatimaAppCompatActivity(), CardAdapterListener {
         mAdapter.toggleSelection(inputPosition)
     }
 
+    private fun showToast(
+        context: Context = applicationContext,
+        @StringRes text: Int,
+        duration: Int = Toast.LENGTH_SHORT,
+    ) = Toast.makeText(context, text, duration).show()
+
```

Comment:
I don't think we should wrap such a simple core Android function, especially not because it hides the fact that we're passing context around and keeping track of context is quite important as using the wrong context can easily break things in weird ways.

Reply:
Or from another point of view we don't use Toast.makeText function many times, avoids using wrong context or duration, and passing 1 argument instead 3 (easier in call points). But imho it is very insignificant change.